<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>20</red>
        <green>22</green>
        <blue>24</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string> ARP Spoofing Detected And Personal Network Scanner</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-multimedia">
    <normaloff>../../../Downloads/ARP.png</normaloff>../../../Downloads/ARP.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.999900000000000</double>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">

QMainWindow {
  background-color: #141618;

}</string>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: #141618;


}</string>
   </property>
   <widget class="QLabel" name="Burger">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
background: rgba(0,0,0,0.0);
}
.QLabel:hover {
  background-color: #FC3666;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Assets/burger.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>501</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel#title{
font-family:Constantia;
font-size: 20px;
background: rgba(0,0,0,0.0);
text-align: center;
color:#5A6272;
}</string>
    </property>
    <property name="text">
     <string> ARP Spoofing Detection And  Network Scanner</string>
    </property>
   </widget>
   <widget class="QLabel" name="exit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
background: rgba(0,0,0,0.0);
}
.QLabel:hover {
  background-color: #FC3666;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Assets/cross.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>901</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="ScanWin">
       <property name="styleSheet">
        <string notr="true"> .QPushButton#ScanWin{

background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;

}
QPushButton#ScanWin:hover {
 background-color: #FC3666;
}</string>
       </property>
       <property name="text">
        <string>Network Scanner</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="PassiveWin">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"> .QPushButton#PassiveWin{
background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}
QPushButton#PassiveWin:hover {
 background-color: #FC3666;
}</string>
       </property>
       <property name="text">
        <string>Passive Detection</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>PassiveWin</zorder>
    <zorder>ScanWin</zorder>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>881</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="PassiveD">
     <widget class="QPushButton" name="Start_Passive">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>221</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QPushButton#Start_Passive{

background-color: #141618; 
border-radius: 1px;
border-bottom: 2px solid #FC3666;
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}
QPushButton#Start_Passive:hover {
 background-color: #FC3666;
}</string>
      </property>
      <property name="text">
       <string>Start Passive Detection</string>
      </property>
     </widget>
     <widget class="QLabel" name="Start_labelP">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>40</y>
        <width>251</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#Start_labelP{
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="OutPut_labelP">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>361</width>
        <height>91</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel#OutPut_labelP{
border-radius: 1px;
 color: rgb(255, 0, 0);
 padding: 20px;
 font-size: 16px;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="maxMacChangesSpinBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>360</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QSpinBox#maxMacChangesSpinBox{
border: 0.5px solid #FC3666;
 color: #F5F5F5;
}</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>4</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="timeWindowSpinBox">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>300</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QDoubleSpinBox#timeWindowSpinBox{
border: 0.5px solid #FC3666;
color: #F5F5F5;
}</string>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="accelerated">
       <bool>false</bool>
      </property>
      <property name="value">
       <double>60.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="MaxMacLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>340</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel#MaxMacLabel{
border-radius: 1px;
 color: #F5F5F5; 
}
</string>
      </property>
      <property name="text">
       <string>Max Mac Changes :</string>
      </property>
     </widget>
     <widget class="QLabel" name="TimeWindowLabel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>280</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">.QLabel#TimeWindowLabel{
border-radius: 1px;
 color: #F5F5F5; 
}
</string>
      </property>
      <property name="text">
       <string>Time Window :</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Newtwork_Scanner">
     <widget class="QPushButton" name="Scan_Button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>190</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  .QPushButton#Scan_Button{

background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}
QPushButton#Scan_Button:hover {
 background-color: #FC3666;
}</string>
      </property>
      <property name="text">
       <string>Start Scan</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>680</width>
        <height>425</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>680</width>
        <height>425</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>680</width>
        <height>425</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QTableWidget{

background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;

}
QTableWidget::item:selected {
        background-color: #c3daf9;
    }

    QHeaderView::section {
     background-color: #141618; 
     border: none;
     border-right: 1px #FC3666;
     border-bottom: 1px solid #FC3666;
     border-radius: 1px;
	color: #F5F5F5;
 	padding: 6px;
 	font-size: 13px;
    }

 
    QHeaderView::section:last {
        border-right: none;
    }

    QHeaderView::section:horizontal {
        height: 40px;
    }





    QTableWidget::item {
        padding: 6px;
		border-bottom: 1px solid #FC3666;
 		font-size: 6px;
    }

   </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
      </property>
      <property name="autoScrollMargin">
       <number>10</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="showDropIndicator" stdset="0">
       <bool>false</bool>
      </property>
      <property name="dragDropOverwriteMode">
       <bool>false</bool>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>135</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>30</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="audio-card">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="Target_Network">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&amp;quot;Enter the target network in CIDR notation (e.g., 192.168.1.0/24). If left blank, the default value 192.168.1.1/24 will be used.&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QLineEdit#Target_Network{


border-radius: 1px;
border: 1px solid #FC3666;
 color: #F5F5F5;



 font-size: 12px;
}
</string>
      </property>
      <property name="text">
       <string>192.168.1.1/24</string>
      </property>
     </widget>
     <widget class="QPushButton" name="export_button">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>290</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  .QPushButton#export_button{

background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}
QPushButton#export_button:hover {
 background-color: #FC3666;
}</string>
      </property>
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>140</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="scan_radio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"> .QRadioButton#scan_radio{
border-radius: 1px;
 color: #F5F5F5;
 font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>Scan Network</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="sniff_radio">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"> .QRadioButton#sniff_radio{
border-radius: 1px;
 color: #F5F5F5;
 font-size: 12px;
}</string>
         </property>
         <property name="text">
          <string>Sniff traffic</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="Filter_Edit">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>40</y>
        <width>171</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QLineEdit#Filter_Edit{
border-radius: 1px;
border: 1px solid #FC3666;
color: #F5F5F5;
font-size: 12px;
}
</string>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Sniffer_Button">
      <property name="geometry">
       <rect>
        <x>701</x>
        <y>190</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">  .QPushButton#Sniffer_Button{

background-color: #141618; 
border-radius: 1px;
border-bottom: 1px solid #FC3666;
 color: #F5F5F5;
 padding: 20px;
 text-align: center;
 text-decoration: none;
 font-size: 16px;
}
QPushButton#Sniffer_Button:hover {
 background-color: #FC3666;
}</string>
      </property>
      <property name="text">
       <string>Start Sniffer</string>
      </property>
     </widget>
     <widget class="QLabel" name="Filter_label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QLabel#Filter_label{
border-radius: 1px;
 color: #F5F5F5;
 font-size: 12px;
}
</string>
      </property>
      <property name="text">
       <string>Filter:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Target_label">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"> .QLabel#Target_label{
border-radius: 1px;
 color: #F5F5F5;
 font-size: 12px;
}
</string>
      </property>
      <property name="text">
       <string>Target IP Address Range:</string>
      </property>
     </widget>
     <zorder>tableWidget</zorder>
     <zorder>Target_Network</zorder>
     <zorder>export_button</zorder>
     <zorder>horizontalLayoutWidget</zorder>
     <zorder>Filter_Edit</zorder>
     <zorder>Sniffer_Button</zorder>
     <zorder>Scan_Button</zorder>
     <zorder>Filter_label</zorder>
     <zorder>Target_label</zorder>
    </widget>
   </widget>
   <widget class="QLabel" name="minim">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">.QLabel {
background: rgba(0,0,0,0.0);
}
.QLabel:hover {
  background-color: #FC3666;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Assets/minimize-sign.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>title</zorder>
   <zorder>Burger</zorder>
   <zorder>exit</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>minim</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ScanWin</tabstop>
  <tabstop>PassiveWin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
